input_output/file_output.py

```diff
@@ -0,0 +1,33 @@
+from supporting_components.graph_io import write_dot
+import subprocess
+
+
+"""
+Before you can use this module you must install graphviz on the terminal to be able to do this:
+    https://stackoverflow.com/questions/1494492/graphviz-how-to-go-from-dot-to-a-graph
+Install instructions:
+    https://www.graphviz.org/download/
+"""
+
+
+def save_graph_as_dot(G, filename):
+    """
+    This method saves a graph in a dot file  .
+    :param G: The graph to be saved.
+    :param filename: The filename in which the graph should be saved.
+    """
+
+    dot_filename = '../output_graphs/dot/' + filename + '.dot'
+    with open(dot_filename, 'w') as g0:
+        write_dot(G, g0)
+
+
+def save_graph_in_png(filename):
+    """
+    This method can use a .dot file and transform it into a png image of the graph.
+    :param filename: The filename of the .dot file without the extension, this will also be the filename for the .png file
+    """
+
+    dot_filename = '../output_graphs/dot/' + filename + '.dot'
+    png_filename = '../output_graphs/png/' + filename + '.png'
```

Comment:
directory doesn't exist error. Creating the folder with __ init __.py works

Reply:
Omdat we output_graphs in gitignore hebben moet je de mappen dot en png inderdaad zelf aanmaken.